src/pages/AddDataPage.tsx

```diff
@@ -44,7 +44,7 @@ const AddDataPage = () => {
 
   const noStoredData = useMemo<boolean>(() => {
     const gestureData = gestures.data;
-    return (
+    return !(
```

Comment:
Small fix in logic in Add Data page so that data can be visible without connecting